.gitignore

```diff
@@ -26,3 +26,4 @@ benchmark_outputs
 node_modules
 product-test-reports
 .vscode/
+lib/trino-parquet/src/main/resources/io/trino/parquet/writer/build.properties
```

Comment:
generated sources should go inside target/

anyway, this is not needed, remove